docs/api-docs/cart-and-checkout/embedded-checkout-overview.md

```diff
@@ -2,38 +2,38 @@
 
 <div class="otp" id="no-index">
 
-### On This Page
-- [How it Works](#how-it-works)
-- [Channels, Sites and Routes APIs](#channels-sites-and-routes-apis)
+### On this page
+- [How it works](#how-it-works)
+- [Channels, sites and routes APIs](#channels-sites-and-routes-apis)
```

Comment:
```suggestion
- [Channels, Sites, and Routes APIs](#channels-sites-and-routes-apis)
```
These should be capitalized since they are the proper names of the APIs